app/models/miq_provision_virt_workflow.rb

```diff
@@ -911,7 +911,11 @@ def ws_template_fields(values, fields, ws_values)
     if src.nil?
       raise _("No source template was found from input data:<%{data}>") % {:data => data.inspect}
     end
-    _log.info "VM Found: <#{src.name}> <#{src.guid}> <#{src.uid_ems}>  Datacenter:<#{src.datacenter_name}>"
+
+    msg = "VM Found: <#{src.name}> <#{src.guid}> <#{src.uid_ems}>  "
```

Comment:
the trailing space at the end of this line should be moved to the conditional `msg += "  Datacenter ..."` below